cypress/e2e/exerciseList.cy.ts

```diff
@@ -0,0 +1,36 @@
+import { exerciseList } from '../intercepts/exerciseList';
+import { meSuperadmin } from "../intercepts/login";
+import { createExercise } from "../intercepts/createExercise";
+import { listCategories } from "../intercepts/categories";
+
+describe("Exercise List Page", () => {
+    beforeEach(() => {
+        meSuperadmin();
+        cy.visit('http://localhost:5173/exercises');
```

Comment:
shoud also add here the exerciseList intercept

Reply:
added